gensim/models/base_any2vec.py

```diff
@@ -309,6 +309,7 @@ def __init__(self, sentences=None, workers=3, vector_size=100, epochs=5, callbac
         self.min_alpha = float(min_alpha)
         self.hs = int(hs)
         self.negative = int(negative)
+        self.num_bits = int(num_bits)
```

Comment:
need to be sure that old model will be loaded successfully (need to modify `load` function) + backward compatibility test (this modification needed only for w2v class I think).